src/Image.php

```diff
@@ -925,7 +880,7 @@ public static function callBase(
         $instance,
         array $arguments
     ) {
-        self::debug($name, $instance, $arguments);
+        Main::debug($name, array_merge(['instance' => $instance], $arguments));
```

Comment:
We could use: `Main::debug($name, ['instance' => $instance, 'arguments' => $arguments)]);` instead of the `array_merge`.